doc/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Requirements for the documentation.
+# These are extra dependencies that aren't required to use sphinx-gallery.
+matplotlib
+pillow
+seaborn
+scipy
+sphinx-gallery

```

Comment:
I know this is needed for the sphinx-gallery examples in binder but this feels a bit weird if you are using this file locally in order to install the requirements needed to build the doc. Personally I don't but still ...

Maybe we can put a requirements file for binder e.g. in binder/requirements.txt and change conf.py?

Reply:
Well this is in doc/ not in the root of the directory. But indeed, I do want to have a better description that this is for the binder environment.

Reply:
You could easily think that the `doc/requirements.txt` is to install the dependencies needed to generate the documentation. If you move it to `doc/binder/requirements.txt` it is a bit harder to be confused.

Reply:
I'm happy to put in a `binder/` folder

Reply:
though FWIW, when I see a file called `doc-requirements.txt` I assume it's the requirements to *run* the documentation, not build it. Most users aren't developers :-)